packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -304,9 +416,22 @@ export const SlackNotifier: Notifier = {
 
   async endMeeting(dataLoader: DataLoaderWorker, meetingId: string, teamId: string) {
     const {meeting, team} = await loadMeetingTeam(dataLoader, meetingId, teamId)
+    const meetingResponses = await getTeamPromptResponsesByMeetingId(meetingId)
+    const standupResponses: Array<{user: User; response: TeamPromptResponse}> = []
```

Comment:
I started attempting to pull out a separate `endStandupMeeting` method, but with the existing interfaces this gets kinda hairy, since we'd need a separate `endStandupMeeting` on _all_ `NotificationIntegrationHelper`s, which for MS teams + mattermost means either 
a) a `NotificationIntegrationHelper.endStandupMeeting` identical to `endMeeting`, or
b) a `Notifier.endStandupMeeting` that calls `NotificationIntegrationHelper.endMeeting` under the hood, and a `NotificationIntegrationHelper.endStandupMeeting` that throws an `unimplemented` error
neither of which seems much better.

I agree that a standup-specific arg being passed to a generic method isn't ideal, but overall, this feature is very experimental, so I'd rather not put too much effort into doing this the "right" way until we start iterating. I think when we add this feature to mattermost + ms teams we'll do whatever refactor makes the most sense here.

Reply:
makes sense! as long as we budget some time to fix it up once it's no longer experimental i'm good to go